charts/bundler/templates/static_config.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-static-data
+  namespace: {{ .Release.Namespace }}
+data:
+  {{ $currentScope := . }}
+  {{ range $path, $_ := .Files.Glob "static/*.json" }}
```

Probably want to pass a user controlled glob here

---

can you nest templates?

---

Not sure what you mean by nest, but you can have this:

```suggestion
  {{ range $path, $_ := .Files.Glob .Values.bundler.staticFileGlob }}
```